automated/android/multinode/tradefed/tradefed-multinode.yaml

```diff
@@ -83,7 +78,7 @@ run:
         # available in the CTS execution scripts.
         - |
             exec_result=0
-            sudo -u testuser env "PATH=${PATH}" ./tradefed-multinode.sh \
```

Comment:
The testuser was introduced for vts, vts didn't work with root for some reasons. I am not sure if it still is the case now. As using root for both will make things simpler, I tend to add +1 for this commit. We can revisit it when we  introduce vts to multinode testdef.